docker-compose.yml

```diff
@@ -73,7 +73,7 @@ services:
       - -c
       - |
         cd /home/site/wwwroot
-        npm install -g azure-functions-core-tools
+        npm install -g azure-functions-core-tools@4.0.6280
```

Comment:
This change should probably be called out in the PR description

Reply:
I believe they made a follow up card to update this once the dependency is fixed

Reply:
Good point, it was called out in a comment but that is less visible unless you hunt for it.